### PR TITLE
ddt removal in ui puppet classes tests

### PR DESCRIPTION
**Pull request by sthirugn**
Tests pending

### CODE REVIEW

File: tests/foreman/ui/test_puppet_classes.py [hunk 1]
```diff
@@ -25,26 +23,23 @@ def test_update_positive(self, description):
...
+                        class_name, param_name, description)
+                    self.assertEqual(
+                        description,
+                        self.puppetclasses.fetch_class_parameter_description(
+                            class_name, param_name)
+                    )
+
     @run_only_on('sat')
-    def test_delete_positive(self, name):
+    def test_delete_positive(self, ):
```
Comment (oshtaier) [inline tests/foreman/ui/test_puppet_classes.py]: `def test_delete_positive(self):`

**Comment by oshtaier** [comment]
APT pending non-blocking comment

**Comment by abalakh** [comment]
ACK pending tests results.

**Comment by sthirugn** [comment]
Comment addressed and rebased.
```
# nosetests tests/foreman/ui/test_puppet_classes.py
..
----------------------------------------------------------------------
Ran 2 tests in 289.675s

OK
```

File: tests/foreman/ui/test_puppet_classes.py [hunk 2]
```diff
@@ -25,26 +23,23 @@ def test_update_positive(self, description):
...
-            )
-            self.assertEqual(
-                description,
-                self.puppetclasses.fetch_class_parameter_description(
-                    class_name,
-                    param_name)
-            )
-
-    @data(*generate_strings_list(len1=8))
+            for description in generate_strings_list(len1=8):
```
Comment (jyejare) [inline tests/foreman/ui/test_puppet_classes.py]: I believe we can increase the length of description here. Usually we will have more than 8 chars in description. Also we can test how the description behaves with long strings.

**Comment by jyejare** [comment]
ACK pending non blocking comment.

File: tests/foreman/ui/test_puppet_classes.py (hunk 2 above)
Comment (sthirugn) [inline tests/foreman/ui/test_puppet_classes.py]: @jyejare ty for the suggestion.  I just reused the old value here.  Lets address this in https://github.com/SatelliteQE/robottelo/issues/2966 to investigate more on why len1=8 was used earlier and we can increase it as necessary.

**Comment by sthirugn** [comment]
Comments addressed, ready for review/merge.

**Comment by elyezer** [comment]
This got 3 ACKs, and all were addressed. Time to merge.

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**